src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4751,7 +4751,11 @@ internal static string GetGetVerbText()
 param(
     [Parameter(ValueFromPipeline=$true)]
     [string[]]
-    $verb = '*'
+    $Verb = '*',
+
+    [Parameter(ValueFromPipeline=$true)]
+    [string[]]
+    $Group
```

Comment:
You can see a enum using sample in [Get/Set-Alias](https://github.com/PowerShell/PowerShell/blob/309bf603f9aff9009dce7e725d42b2d4f99f58c9/src/Microsoft.PowerShell.Commands.Utility/commands/utility/WriteAliasCommandBase.cs#L47)
The enum could be placed [ Verbs.cs](https://github.com/PowerShell/PowerShell/blob/309bf603f9aff9009dce7e725d42b2d4f99f58c9/src/System.Management.Automation/utils/Verbs.cs)